charts/harmony-chart/Chart.yaml

```diff
@@ -17,24 +17,24 @@ dependencies:
 # This is just info for the "helm dependency update" command, which will update the ./charts/ directory when run, using
```

Comment:
@Ian2012 we need to bump the chart version to a new one (line 8 of this file). I would suggest 0.8.0 as the new version